modules/publication/jsx/uploadForm.js

```diff
@@ -6,6 +6,10 @@ import {
   FormElement,
   TextboxElement,
 } from 'jsx/Form';
+import {withTranslation} from 'react-i18next';
+import i18n from 'I18nSetup';
+import hiStrings from '../locale/hi/LC_MESSAGES/publication.json';
+i18n.addResourceBundle('hi', 'publication', hiStrings);
```

Comment:
This shouldn't be here. It should be in the entry point of the page or (if that doesn't work) componentDidMount/useEffect of the componenet